freqtrade/pairlist/StaticPairList.py

```diff
@@ -38,4 +48,8 @@ def filter_pairlist(self, pairlist: List[str], tickers: Dict) -> List[str]:
         :param tickers: Tickers (from exchange.get_tickers()). May be cached.
         :return: new whitelist
         """
-        return self._whitelist_for_active_markets(self._config['exchange']['pair_whitelist'])
+        if self._pairlist_pos != 0:
+            raise OperationalException(f"{self.name} can only be used in the first position "
+                                       "in the list of Pairlist Handlers.")
```

Comment:
Shouldn't this be in the constructor? 

i guess- i'd expect it to fail when PairlistManager is created - not when i try to get a pairlist ...